arch/sparc/src/common/up_internal.h

```diff
@@ -175,6 +182,12 @@ extern uint32_t _bmxdupba_address;   /* BMX register setting */
  * functions prototyped in include/nuttx/arch.h.
  */
 
+/* Atomic modification of registers */
+
+void modifyreg8(unsigned int addr, uint8_t clearbits, uint8_t setbits);
+void modifyreg16(unsigned int addr, uint16_t clearbits, uint16_t setbits);
+void modifyreg32(unsigned int addr, uint32_t clearbits, uint32_t setbits);
+
```

Comment:
Can we move this block before `/* Common Functions *********************************************************/`, just for consistency with other similar changes?

Reply:
But `/* Common Functions *********************************************************/` is before this block in all other place:
 https://github.com/apache/incubator-nuttx/pull/5720/files?file-filters%5B%5D=.h&show-deleted-files=true&show-viewed-files=true#diff-206bf33ecfcef35eb3b5bf095ebc866558dc44a86ec3bfcf099295c304c944e3R181-R193
https://github.com/apache/incubator-nuttx/blob/master/arch/xtensa/src/common/xtensa.h#L217-L228

Reply:
Ok